src/components/footer/Footer.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useStaticQuery, graphql } from 'gatsby';
 import Img from 'gatsby-image';
 import { FormattedMessage } from 'react-intl';
-
+// @ts-ignore
```

Comment:
why you use this comment? there aren't any errors

Reply:
I have used it because there is an error "can not find path", though everything works perfectly well, it's a common problem for the whole project, meybe, there is something wrong with the settings of my environment. I will try to fix it.